301-eventHub-create-authrule-namespace-and-eventHub/README.md

```diff
@@ -0,0 +1,5 @@
+# 301-eventHub-create-authrule-namespace-and-eventHub
```

Comment:
Complexity of a template for our service is defined by the type of resource being created and not just the number. Besides there a lot going on in this template. eg: some resources need to be created serially and that needs the correct use of dependson property. A lot of customers have reported this issue and it needs to addressed via a template.